src/core/definitions.jl

```diff
@@ -20,26 +20,20 @@ const JuMPVariableArray = JuMP.Containers.DenseAxisArray{JuMP.VariableRef}
 const JuMPParamArray = JuMP.Containers.DenseAxisArray{PJ.ParameterRef}
 const DenseAxisArrayContainer = Dict{Symbol, JuMP.Containers.DenseAxisArray}
 
-@enum BUILD_STATUS begin
-    BUILT = 0
+IS.@scoped_enum BUILD_STATUS begin
```

Comment:
Our convention has been to make the enum type PascalCase and the enum values ALL_CAPS, like global constants.

Reply:
We need to add this to the style guide then. I will make the change

Reply:
addressed in c6af5b0